modules/google-cloud-scc-configuring-acs.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-google-cloud-scc.adoc
+:_module-type: PROCEDURE
+[id="google-cloud-scc-configuring-acs_{context}"]
+= Configuring {product-title}
```

Comment:
Same comment as other integration PRs about making this a more specific/descriptive heading